### ISSUE TITLE
Could not find auto-linked framework

### ISSUE THREAD

**Issue by christinasund** [kylef/Mockingjay#114]
I have successfully installed via Cocoapods, imported the module in my XCTest class, and added methods. When I try and run my tests I see the below error. I have tried to import the project manually with no luck, have the framework in my linked frameworks for the test target. Any ideas?

![image](https://user-images.githubusercontent.com/18584125/60356588-38da2a80-9986-11e9-8c62-c3a2511e6bd6.png)

Thanks

**Comment by kylef** [comment]
@christinasund are you able to share your Podfile (or atleast the important parts, you can redact target names and remove other dependencies). I'd check that Mockingjay is inside the test target itself, for example:

```ruby
source 'https://github.com/CocoaPods/Specs'

use_frameworks!
inhibit_all_warnings!

target 'Palaver' do
  pod 'PalaverKit', :git => 'https://github.com/kylef/PalaverKit.git'
  pod 'Bugsnag'
  pod 'Commander'

  target 'PalaverTests' do
    inherit! :search_paths

    pod 'Mockingjay', '3.0.0-alpha.1'
  end
end
```

**Comment by christinasund** [comment]
@kylef Yes, no problem. I have added Mockingjay to the test target only

`# Uncomment the next line to define a global platform for your project
# platform :ios, '9.0'

target 'ios' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for ios

end

target 'macOS' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for macOS

end

target 'OSXTestHost' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for OSXTestHost

end

target 'SwiftTestbed' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for SwiftTestbed

end

target 'TestHost' do
  # Comment the next line if you don't want to use dynamic frameworks
  use_frameworks!

  # Pods for TestHost
  pod 'Mockingjay'
end
`

**Comment by kylef** [comment]
This can depend a bit on the configuration of your project, it may be that `TestHost` is an application which has a separate test target (perhaps `TestHostTests`) which is injected into it.

Perhaps we can confirm that with getting a full list of the targets, for example:

```
$ xcodebuild -list
Information about project "Palaver":
    Targets:
        Palaver
        PalaverTests
        PalaverUITests
```

You may need to pass `-workspace <filename for xcode workspace>`.

**Comment by christinasund** [comment]
I think you might be right, but I do not see this other test target in my Podfile, what do you suggest?

```
Information about project "swift":
    Targets:
        swift-tests-ios
        swift-tests-osx
        swift-tests-tvos
        ios
        tvos
        watchOS
        macOS
        TestHost
        OSXTestHost
        TVOSTestHost
        swift-tests-ios-crash
        SwiftTestbed
        Core
        AppData
        Attribution
        Autotracking
        Collect
        Connectivity
        ConsentManager
        Crash
        DataSource
        DefaultsStorage
        Delegate
        DeviceData
        DispatchQueue
        FileStorage
        Lifecycle
        Logger
        RemoteCommands
        TagManagement
        VolatileData
        VisitorProfile

    Build Configurations:
        Debug
        Release

    If no build configuration is specified and -scheme is not passed then "Release" is used.

    Schemes:
        ios
        macOS
        OSXTestHost
        SwiftTestbed
        swift-tests-osx
        swift-tests-tvos
        AppData
        Attribution
        Autotracking
        Collect
        Connectivity
        ConsentManager
        Core
        CrashModule
        DataSource
        DefaultsStorage
        Delegate
        DeviceData
        DispatchQueue
        FileStorage
        Lifecycle
        Logger
        RemoteCommands
        TagManagement
        VisitorProfile
        VolatileData
        TestHost
        tvos
        TVOSTestHost
        watchOS
```

**Comment by kylef** [comment]
I think you could see what it hooked up in the scheme, it might show you which test target is injected into a host app scheme:

On the top menu:

- Product -> Scheme -> Select Scheme
- Product -> Scheme -> Edit Scheme, then select the "test" tab on left side.

<img width="954" alt="Screenshot 2019-07-01 at 10 40 03" src="https://user-images.githubusercontent.com/44164/60426598-dfc00000-9bec-11e9-8104-865c19f63550.png">

In the reverse, you can see which target is put into which host application using the project/target views of Xcode:

<img width="741" alt="Screenshot 2019-07-01 at 10 43 53" src="https://user-images.githubusercontent.com/44164/60426757-3594a800-9bed-11e9-9cef-869e35b2da74.png">
